src/query.ts

```diff
@@ -507,6 +510,45 @@ export class Query<T extends Model> {
     return _.pick(instance, Object.keys(getModelAssociations(this.model))) as T;
   }
 
+/**
+ * Returns the associations that correlate to the current include options
+ * @param {boolean} required? Whether or not to only get required associations (associations that are not nullable).
+ * @return {any[]} An array of association descriptors.
+ */
+  private getIncludedAssociations(required?: boolean): any[] {
```

Comment:
* In that case, you can just prefix it with `_` and explain the code doc that it's unused.
* I'd say let's just have a FIXME note and keep it as any. Probably not adding too much value if it's going to be a minimal definition and any interface changes will be caught during the regression tests you wrote